scikitplot/decomposition.py

```diff
@@ -156,14 +166,36 @@ def plot_pca_2d_projection(clf, X, y, title='PCA 2-D Projection', ax=None,
     classes = np.unique(np.array(y))
 
     colors = plt.cm.get_cmap(cmap)(np.linspace(0, 1, len(classes)))
-
+    
+    vectors=np.transpose(clf.components_[0:2, :])
```

Comment:
Style: Add spaces around equals sign.

Also, not sure about this but can `0:2` be written as simply `:2`?

Reply:
Sry, Bad habit on indexing, I'll fix that.